functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -70,6 +70,8 @@ private BuiltProject executeGoals(String... goals) {
                     .setProperties(asProperties(buildConfigurator.getSystemProperties()))
                     .skipTests(buildConfigurator.isSkipTestsEnabled())
                     .setMavenOpts(buildConfigurator.getMavenOpts())
+                    .setWorkingDirectory(buildConfigurator.getWorkingDirectory())
+                    .setAlsoMake(true)
```

Comment:
> I think also-make should be enabled by default if we specify only particular projects to build. Otherwise false with the ability to enable (? do we really need that then).

Yes it should be ideally like this. 

> I will make new issue for that, let's not spoil this PR if it does not have tremendous impact on the build time. 
I think it doesn't have much impact we can create new issue for it to work on it later.